src/main/java/org/jenkinsci/plugins/workflow/actions/ErrorAction.java

```diff
@@ -53,6 +54,19 @@ public ErrorAction(Throwable error) {
      * an equivalent that captures the same details but serializes nicely.
      */
     private boolean isUnserializableException(Throwable error) {
+        if (error == null) {
+            // This shouldn't happen.
+            return false;
+        }
```

Comment:
I can expect that `error` never be `null` as `ErrorAction` is always called like
```
if (outcome.isFailure()) {
    head.addAction(new ErrorAction(outcome.getAbnormal()));
}
```

On the other hand, I can't mark `error` with `NonNull` annotation as `getAbnormal` can be `null` if `isFailure()` is `false`
and static code analysis by FindBug can't know `error` never be `null` in this context.